hack/verify-chart-version.sh

```diff
@@ -31,7 +31,7 @@ fi
 
 git fetch jetstack "${PULL_BASE_SHA}:refs/remotes/jetstack/pull-base"
 
-SCRIPT_ROOT=$(dirname "${BASH_SOURCE}")/..
+SCRIPT_ROOT="$(dirname "${BASH_SOURCE}")/.."
```

Comment:
Not sure if the two changes in this file are intentional?

Reply:
Yup, intentional